packages/docs/src/README.md

```diff
@@ -26,7 +30,17 @@ const app = Vue.createApp(App)
 app.use(Vuelidate)
 ```
 
-Then in your component you can create a `validations` method and define your validation rules.
+Now that Vuelidate is registered as a global plugin, create a `validations` method inside your component and define your validation rules.
+
+First, import the validators that you want to use from `@vuelidate/validators`.
+
+```js
+import { required } from '@vuelidate/validators'
+```
+
+Next, add the `validations` method, it should return an object that matches the structure of your state. In this example, we have a field `name`, which is declared inside of the component's `data()`.
+
+Notice that in the `validations` method we declare the same structure, and the `name` property specifies the validators that we want to apply to it - in this case, a `required` validation.
```

Comment:
`in the validations method we declare the same structure` this is slightly unclear to me. 

I think this would be best explained visually.
1) Show a combination of data (ideally with an object inside with their own properties) + a computed property
2) Show how the above local state is then available on `this`.
3) Show how that structure maps 1:1 to what we return from `validations`.